dao/searchDao.js

```diff
@@ -1,11 +1,20 @@
 const db = require('./db');
+const sqliteDatabase = require('better-sqlite3');
```

Comment:
Do you really want to `require` (i.e. pull into memory) the whole `better-sqlite3` module in the production environment? If you can make an interface and set its implementation based on which environment, that would be much better.